subprojects/packagefiles/theora/lib/meson.build

```diff
@@ -199,14 +199,19 @@ libtheoraenc = library(
   install: true,
 )
 
+# In 1.1.1 the source pkg-config specifies an incorrect
+# dependency on theoradec which is relied on by GStreamer.
+# Otherwise th_version_string is invisible according to the
+# version scripts/.def files.
 pkg.generate(
   libtheoraenc,
+  requires: libtheoradec,
   description: 'Theora video codec (encoder)',
   version: meson.project_version(),
 )
 
 theoraenc_dep = declare_dependency(
-  link_with: libtheoraenc,
```

Comment:
AFAIK, an alternative to this would be to add libtheoradec_dep to dependencies.

Reply:
If the include directories are the same, then this is simpler.

Reply:
They have the same incdir, so I think this is fine like this.